lib/Service/Config/ConfigLoader.hs

```diff
@@ -73,16 +73,19 @@ loadDSWConfig applicationConfigFile buildInfoFile = do
         , _appConfigDatabasePassword = password
         }
     loadAppConfigMessaging configParser = do
-      host <- get configParser "Messaging" "host"
-      port <- get configParser "Messaging" "port"
-      username <- get configParser "Messaging" "username"
-      password <- get configParser "Messaging" "password"
+      mbMqEnabled <- getOptional configParser "Messaging" "enabled"
```

Comment:
To be consistent with the naming in the rest of the app, I would change the name from `mbMqEnabled` to `mMqEnabled`